app/scripts/services/Courts.svc.js

```diff
@@ -5,16 +5,17 @@ angular.module('yourStlCourts').factory('Courts', function ($resource, $q) {
   var courts;
```

Comment:
Is this a cache? We should really be doing this in the browser caching layer, not ourselves.